src/gl/vbo_mesh.js

```diff
@@ -12,6 +12,7 @@ export default class VBOMesh  {
         this.gl = gl;
         this.vertex_data = vertex_data; // typed array
         this.vertex_layout = vertex_layout;
+        this.vertex_elements = options.vertex_elements;
         this.buffer = this.gl.createBuffer();
```

Comment:
Maybe we should rename this `vertex_buffer` to distinguish, since we now have a second (element) buffer?